setup.cfg

```diff
@@ -18,21 +18,24 @@ package_dir =
 packages = find:
 python_requires = >=3.6
 install_requires =
-  numpy == 1.21
-  scipy == 1.7
-  matplotlib == 3.5
-  numba == 0.55
-  ipykernel == 6.9   # Allows this conda environment to show up automatically in Jupyter Lab
-  watermark == 2.3    # Generates a summary of package version for use inside Jupyter Notebooks
+    numpy == 1.21
+    scipy == 1.7
+    matplotlib == 3.5
+    numba == 0.55
+    ipykernel == 6.9  # Allows this conda environment to show up automatically in Jupyter Lab
+    watermark == 2.3  # Generates a summary of package version for use inside Jupyter Notebooks
```

Comment:
I think these should be dev dependencies, since someone installing mcmc will not be using them,

Reply:
I'll leave them here for now